### ISSUE TITLE
Possibility to force a schedule via automation

### ISSUE THREAD

**Issue by rhoddan** [arthurdent75/SimpleScheduler#122]
Thanks for a great add-on. Good that there is a possibilty to enable/disable a schedule via an automation.

I wish to be able to also turn on / turn off a schedule same way (i.e for example run every normal settings via simple schedule but in some cases I also want to fire an turn on/dim or other command via an automation rule)

**Comment by rhoddan** [comment]
Yes, I know that it’s possible to enable/disable. I mean that if you have configure a bunch of lights etc and normally use the simple scheduler to turn on/off, in some cases you want to turn on/off these lights via an automation (not only enable/disable). The reason for that could be to send a turn off if my local lux sensor has reach a limit.

**Comment by arthurdent75** [comment]
> I mean that if you have configure a bunch of lights etc and normally use the simple scheduler to turn on/off, in some cases you want to turn on/off these lights via an automation

The solution, as I said, is to create two script (one for the "ON scenario" and one for "OFF scenario" and then call the scripts from the scheduler. This way you can also call the scripts from anywhere in Home Assistant, including automation.
